_source/user-guide/integrations/status-page.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: article
+title: Create and integration with StatusPage
+permalink: /user-guide/integrations/status-page.html
+tags:
+  - alerts
+  - endpoints
+  - integrations
+contributors:
+  - rhershenhoren 
+  - yberlinger
+---
+
+
+Use this StatusPage integration to notify StatusPage when there are service degradations that impact your platform. 
+
+To implement the integration, you will:
+
+1. Create a pair of alerts: 
+   + An alert configured with a condition to trigger a StatusPage update
+   + An alert configured with the opposite trigger condition, to update the StatusPage when the conditions return to normal
+
+2. Create a pair of custom endpoints to update a specific component, then attach each endpoint to the relevant alert:
+   + An endpoint to update the component with the triggered status 
+   + An endpoint to update the component with the status **_Operational_** 
+
+
+
+
+#### Creating a custom endpoint to update StatusPage components
+
+<div class="tasklist">
+
+##### Allow firewall access
+
+See [Custom Endpoints](/user-guide/integrations/custom-endpoints.html) for general instructions on setting up firewall access for Logz.io notifications.
+
+This may include whitelisting Logz.io IPs and/or creating a verification token.
+
```

Comment:
This is not needed.
StatusPage does not have a limitation on IP

Reply:
resolved